### PR TITLE
added PonyUserDatastore & PonyDatastore for Pony ORM

### PR DESCRIPTION
**Pull request by ET-CS**
Pony ORM datastore added

### CODE REVIEW

**Comment by mattupstate** [comment]
If you merge upstream changes and update the tests I might consider this.

**Comment by willzfarmer** [comment]
What's the status on this? I'd love to use it for my project since we use PonyORM elsewhere...

What needs to be done to make this merge-able?

**Comment by jirikuncar** [comment]
@willzfarmer can you update tests?

File: flask_security/datastore.py [hunk 1]
```diff
@@ -206,6 +217,36 @@ def find_role(self, role):
         return self.role_model.query.filter_by(name=role).first()
 
 
+class PonyUserDatastore(PonyDatastore, UserDatastore):
+    """A Pony ORM datastore implementation for Flask-Security.
+    """
+    def __init__(self, db, user_model, role_model):
+        PonyDatastore.__init__(self, db)
+    	UserDatastore.__init__(self, user_model, role_model)
```
Comment (jirikuncar) [inline flask_security/datastore.py]: please use 4 spaces for indentation

**Comment by jirikuncar** [comment]
Merged via #617.

**Pull request closed without merging**